lib/index.js

```diff
@@ -24,10 +24,10 @@ class Oppsy extends Events.EventEmitter {
           psup: ProcessMonitor.uptime,
           psmem: ProcessMonitor.memoryUsage,
           psdelay: ProcessMonitor.delay,
-          requests: this._networkMonitor.requests,
-          concurrents: this._networkMonitor.concurrents,
-          responseTimes: this._networkMonitor.responseTimes,
-          sockets: this._networkMonitor.sockets
+          requests: this._networkMonitor.requests.bind(this._networkMonitor),
```

Comment:
@Marsup How can i bind networkMoitor context for those func which is getting executed via Items.parallel. 

Give me an idea, i will make appropriate changes.